code/modules/mob/dead/observer/freelook/marker/necrovision.dm

```diff
@@ -17,7 +17,16 @@
 		if (!istype(source))
 			sources -= source
 			continue
-		for(var/t in get_datum_visible_turfs(source))//source.get_visualnet_tiles(visualnet))
+
+
+		var/list/visible = get_datum_visible_turfs(source)
```

Comment:
Return an empty list instead? null perhaps? this works, but it’s pretty weird

Reply:
returning an empty list is valid data, can't use that for an error code

Reply:
use list.len then